### PR TITLE
regen the analysis server library

### PR DESCRIPTION
**Pull request by devoncarew**
- regen the analysis server library

I left out the analytics methods, since those changes are in progress in a separate PR.

@alexander-doroshko

### CODE REVIEW

**Comment by alexander-doroshko** [comment]
Hm, looks like this is not ready to start playing with. Implementations in `RemoteAnalysisServerImpl` are empty

**Comment by devoncarew** [comment]
The regen only build the interface in `generated/AnalysisServer.java` - everything else is hand-written. I'm happy to fill out more if it, but probably wouldn't get a chance to do so today...

**Comment by alexander-doroshko** [comment]
Sure, we are not in a hurry.

**Comment by devoncarew** [comment]
@alexander-doroshko, updated! Sorry for the formatting diffs in `utilities/RequestUtilities.java`...

@bwilkerson, can you take a look at the protocol impl? Parts are hand-written - I believe it matches the spec - but another pair of eyes would be good.

File: Dart/thirdPartySrc/analysisServer/com/google/dart/server/generated/AnalysisServer.java [hunk 1]
```diff
@@ -536,7 +568,7 @@
 
   /**
    * Remove the given listener from the list of listeners that will receive notification when new
-   * analysis results become available.
+     * analysis results become available.
```
Comment (bwilkerson) [inline Dart/thirdPartySrc/analysisServer/com/google/dart/server/generated/AnalysisServer.java]: extra indent?

File: Dart/thirdPartySrc/analysisServer/com/google/dart/server/internal/remote/utilities/RequestUtilities.java [hunk 2]
```diff
@@ -145,25 +146,33 @@ public static JsonElement buildJsonElement(Object object) {
...
+    } else if (object instanceof ImportedElements) {
+      return ((ImportedElements)object).toJson();
     }
     throw new IllegalArgumentException("Unable to convert to JSON: " + object);
   }
 
   /**
    * Generate and return a {@value #METHOD_ANALYSIS_GET_ERRORS} request.
-   * 
+   * <p>
```
Comment (bwilkerson) [inline Dart/thirdPartySrc/analysisServer/com/google/dart/server/internal/remote/utilities/RequestUtilities.java]: I thought `pre` forced a line break (making the `p` unnecessary), but perhaps I'm misremembering.

File: Dart/thirdPartySrc/analysisServer/com/google/dart/server/generated/AnalysisServer.java (hunk 1 above)
Comment (devoncarew) [inline Dart/thirdPartySrc/analysisServer/com/google/dart/server/generated/AnalysisServer.java]: fixed; not sure what this was from

**Pull request merged**